pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -26,47 +26,63 @@ import (
 	"path/filepath"
 	"strings"
 
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/log"
 	"kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/api"
+
 	util "kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/util"
 )
 
 const (
 	isUnusable             string = "no"
+	isRequired             string = "require"
 	nodeLabellerVolumePath        = "/var/lib/kubevirt-node-labeller/"
+
+	supportedFeaturesXml = "supported_features.xml"
+	x86PenrynXml         = "x86_Penryn.xml"
```

Comment:
Any reason to have this here? I see it only in tests.

Reply:
Thanks! :smile: 
Indeed `x86PenrynXml` is used only by `cpu_plugin_test.go` and therefore moved to there :)